internal/cmd/manager/instance/run/lifecycle/run.go

```diff
@@ -227,10 +228,10 @@ func configureStreamingReplicaUser(tx *sql.Tx) (bool, error) {
 }
 
 // configurePgRewindPrivileges ensures that the StreamingReplicationUser has enough rights to execute pg_rewind
-func configurePgRewindPrivileges(majorVersion int, hasSuperuser bool, tx *sql.Tx) error {
+func configurePgRewindPrivileges(majorVersion semver.Version, hasSuperuser bool, tx *sql.Tx) error {
```

Comment:
and here we change the signature to consume a semver but only require the major. Not an improvement IMO, except perhaps we can use `unit64` better, as would be done in Semver